icicle-compiler/src/Icicle/Sea/FromAvalanche/State.hs

```diff
@@ -67,19 +66,19 @@ import           P
 ------------------------------------------------------------------------
 
 data SeaProgramAttribute = SeaProgramAttribute {
-    stateAttributeName  :: Int
-  , stateAttribute      :: Source.Attribute
+    stateInputIndex     :: Int
+  , stateInputId        :: InputId
```

Comment:
it might be worth rolling the `inputIndex` and `inputId` into a `SeaInputId` because whenever we refer to an input in C, we usually need the index. Not really necessary - just a suggestion. Anyway I'm happy `staetAttributeName` is gone